.ci/deps.node_js.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e -x
+
+# NPM commands
+ALEX=$(which alex || true)
```

Comment:
move into a new `.ci/deps.alex.sh`, as it is probably useful from generic images